test/services/storage/processor.spec.ts

```diff
@@ -160,6 +160,23 @@ describe('Storage services: Events Processor', () => {
         expect(billingPlan?.tokenAddress).to.be.eql(token)
         expect(billingPlan?.period).to.be.eql(new BigNumber(billingEvent.returnValues.period))
       })
+      it('should remove billing plan on price 0', async () => {
+        // Create new offer and billing plan
+        const offer = await Offer.create({ provider })
+        const billing = await BillingPlan.create({ offerId: offer.provider, period: 99, price: 1, tokenAddress: token, rateId: 'rbtc' })
+        expect(offer).to.be.instanceOf(Offer)
+        expect(billing?.price).to.be.eql(new BigNumber(1))
+        expect(billing?.tokenAddress).to.be.eql(token)
+        expect(billing).to.be.instanceOf(BillingPlan)
+
+        // should remove billing price
+        billingEvent.returnValues.price = 0
```

Comment:
Hmm sorry did not catch this in the Review where you were adding these tests. 

Can you please create the `billingEvent` in every test? This smells like a problem, when you will forget to change something in some other tests and it will start magically fails. Or move the instantion of the `billingEvent` to `beforeEach` block...

Reply:
Done